### PR TITLE
Add an avail get_data test.

### PR DESCRIPTION
**Pull request by jshaughn**
It just tests an extra code path that we didn't have.

@pilhuhn please review.

### CODE REVIEW

**Comment by josejulio** [comment]
Rubocop strikes again!
@jshaughn could you please check?

**Comment by josejulio** [comment]
Travis is complaining about the cassette **metrics_services**/Templates/Mixed_metrics/Should_requests_raw_data_for_multiple_metrics.yml

I'm not really sure if we should keep metrics_0_0_16 and metrics_services. 
It seems that we are using them both alike.
// cc: @pilhuhn @rubenvp8510

**Comment by rubenvp8510** [comment]
I think at this point we can remove metrics_0_0_16 and leave only metric_services.

This was for the testing  of transition from old versions of metrics to metrics 0.0.16 (mostly endpoint changes). but because now metric_services test against  recent versions of metrics, this doesn't make sense anymore.

**Comment by pilhuhn** [comment]
I think it is ok to treat metrics016 and -services as the same. 
We still need to keep metric08 as this is what is in openshift 3.3 and earlier

**Comment by josejulio** [comment]
I'll fix that and post a PR soon.

File: spec/integration/metric_spec.rb [hunk 1]
```diff
@@ -135,6 +135,7 @@
           id1 = SecureRandom.uuid
           id2 = SecureRandom.uuid
           id3 = SecureRandom.uuid
+          id4 = SecureRandom.uuid
 
           ids = [id1, id2, id3]
           bindings = { id1: id1, id2: id2, id3: id3 }
```
Comment (josejulio) [inline spec/integration/metric_spec.rb]: Could you please add id4 to this hash (`bindings`)?
It will help VCR do its replace-template-magic-thing.

**Comment by jshaughn** [comment]
@josejulio, This dumb PR is finally green :)  Please make it go away unless you see something else that needs to be fixed.  Thanks, Jay

**Comment by josejulio** [comment]
Thank you!

**Pull request merged**